components/NewSpendingForm.js

```diff
@@ -11,7 +11,10 @@ export default function NewSpendingForm({onAddSpending}) {
     const data = Object.fromEntries(formData);
 
     onAddSpending(data);
-    router.push("./spendings");
+
+    const {category} = data;
+    alert("Your spending has been successfully added!");
```

Comment:
If possible don't use alert. Better bulid a modal to give the user your feedback.

Reply:
Good point, Stefan. I'll try to add it to my next PR. Thanks a bunch for the feedback!